### ISSUE TITLE
doesn't catch - %>  ( vs -%> ) .  Could be a suggestion 

### ISSUE THREAD

**Issue by moorthi07** [RyanZim/EJS-Lint#32]
I had this error thrown by ejs  which ejs-lint couldn't detect.
```

err SyntaxError: missing ) after argument list in C:\model.ts while compiling ejs

If the above error is not helpful, you may want to try EJS-Lint:
https://github.com/RyanZim/EJS-Lint
```
for a code like,

` <%= ":" _ %>`

First of all it should not throw an error as this code has both <% and %>. But this ejs detects this space (' ') between _ %> and throws an error.

It could be   a meaning full error , may be its missing a closing '%>' - would be more appropriate. 

 But it was very hard for me to locate this little space in the large file!

**Comment by RyanZim** [comment]
I guess this couldn't hurt to add if you'd like to.
